packages/server/src/api/index.ts

```diff
@@ -488,13 +488,11 @@ router.delete('/user/article_series/:id/articles', async (req, res) => {
  */
 router.post('/user/article_series', async (req, res) => {
     // title is required
-    if (!('title' in req.body)) {
-        res.status(400).send({ error: 'Title is required' });
-        return;
-    }
 
-    if (!req.body.title) {
-        res.status(400).send({ error: 'Title is required' });
+    const response = titleValidator.safeParse({ title: req.body.title });
```

Comment:
Simply pass the req.body . it does contain title

Reply:
I tried that, it always puts out "Title must be a string" even if it is passed in as a string

Reply:
Try this one 

```js
export const titleValidator = z.object({
    title: z
        .string()
        .min(1, {
            message: "Title must be at least 1 character long",
        })
        .or(z.undefined(), {
            message: "Title is required",
        }),
});
```

Reply:
Still same thing but now "invalid input"

Reply:
Got it, was passing it wrong